src/themes/uswitch/theme.json

```diff
@@ -716,14 +716,21 @@
   },
 
   "toggleSwitch": {
-    "outer": {
-      "variant": {
-        "checked": {
-          "backgroundColor": "black"
-        },
-        "unchecked": {
-          "backgroundColor": "grey"
-        }
+    "checked": {
+      "backgroundColor": "black",
+      "borderColor": "black",
+      "knob": {
```

Comment:
this will be adding another CSS rule targeting `knob {}` as it will be treated as a child selector

Reply:
Okay, is it something worth changing?

Reply:
Probably - it's not bad in this case, but if it ends up being built on or copied into another component, that wouldn't be great